session.go

```diff
@@ -109,13 +109,15 @@ func (h *stmtHistory) clone() *stmtHistory {
 }
 
 type session struct {
-	txn     kv.Transaction // Current transaction
-	args    []interface{}  // Statment execution args, this should be cleaned up after exec
-	values  map[fmt.Stringer]interface{}
-	store   kv.Storage
-	sid     int64
-	history stmtHistory
-	initing bool // Running bootstrap using this session.
+	txn        kv.Transaction // Current transaction
+	args       []interface{}  // Statment execution args, this should be cleaned up after exec
+	values     map[fmt.Stringer]interface{}
+	store      kv.Storage
+	sid        int64
+	history    stmtHistory
+	initing    bool                   // Running bootstrap using this session.
+	debugInfos map[string]interface{} // Vars for debug and unit tests.
```

Comment:
I prefer using a callback interface like DDL Callback for more easily customizable.